docs/pages/blog/mui-x-v5.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Introducing MUI X v5.0
+description: We are excited to introduce MUI X v5.0.0!
+date: 2021-11-12T00:00:00.000Z
+authors:
+  ['oliviertassinari', 'm4theushw', 'flaviendelangle', 'DanailH', 'alexfauquette']
+card: true
+---
+
+We are excited to introduce [MUI X v5.0.0](https://github.com/mui-org/material-ui-x/releases/tag/v5.0.0)!
```

Comment:
Yes, I agree, when I was preparing the content a couple of times I felt weird talking about MUI X and DataGrid because at the moment they are the same.